scripts/lhelper.sh

```diff
@@ -60,9 +60,9 @@ main() {
   # Not using $(lhelper activate lite-xl) to support CI
   source "$(lhelper env-source lite-xl)"
 
-  lhelper install freetype2
-  lhelper install sdl2 2.0.14-wait-event-timeout-1
-  lhelper install pcre2
+  if [[ "$OSTYPE" == "msys" ]]; then
+    lhelper install pcre2
+  fi
```

Comment:
SDL2, PCRE2 and Freetype2 are now directly build via meson, only PCRE2 needs it installed bcause of incompatibilities